genesyscloud/telephony_providers_edges_phone/genesyscloud_telephony_providers_edges_phone_proxy.go

```diff
@@ -152,20 +154,27 @@ func (p *phoneProxy) assignUserToStation(ctx context.Context, userId string, sta
 
 // getAllPhonesFn is an implementation function for retrieving all Genesys Cloud Phones
 func getAllPhonesFn(ctx context.Context, p *phoneProxy) (*[]platformclientv2.Phone, error) {
+	log.Printf("Entering the getAllPhonesFn method to retrieve all of the phone ids for export")
 	var allPhones []platformclientv2.Phone
+	const pageSize = 100
+	const sortBy = "id"
+
+	phones, response, err := p.edgesApi.GetTelephonyProvidersEdgesPhones(1, pageSize, sortBy, "", "", "", "", "", "", "", "", "", "", "", "", nil, nil)
```

Comment:
The original call was missing the sortBy "id" field.  The sortBy field is what lets us bypass the search field.  If this is not set, I suspect this can cause problems because if you are exporting more than 10K phones the export is going to fail before we even get all of the ids